packages/web-frontend/cypress/e2e/dashboardReports.spec.js

```diff
@@ -36,7 +36,7 @@ const urlToRouteRegex = url => {
     throw new Error(`'${url}' is not a valid report url: it must contain a 'report' query param`);
   }
 
-  return new RegExp(`report.*\\WisExpanded=true&.*itemCode=${itemCode}[&$]`);
+  return new RegExp(`report/.*\\?(.*&|)isExpanded=true&(.*&|)itemCode=${itemCode}(&|$)`);
```

Comment:
I've seen you change this a few times so I'm not going to bother trying to parse it, will trust you've got it sorted now :-)

Reply:
Haha yeah this time I tested it more thoroughly locally! It would help if I would have added unit tests to test variations for this in the first place, but I thought no point testing the test code 😁